### DISCUSSION TITLE
Regarding to the size of message_k

### DISCUSSION THREAD

**Discussion by LeeTroy** [DMTF/libspdm#858] [Q&A] [answered]
https://github.com/DMTF/libspdm/blob/d9a07ede65a818236411abfe5d580e9e49024f4e/include/internal/libspdm_common_lib.h#L236-L242
Why does `temp_message_k` declare with medium size buffer where the `message_k` declares with large size buffer? I've met the following error: `libspdm_append_managed_buffer 0x1e6 fail, rest 0x7a only`, but it will be resolved if I revise medium to large buffer.

**Comment by steven-bellock** [comment, marked as answer]
Yeah we should probably change that to a large managed buffer. Please file an issue.
